### PR TITLE
Redirect user to browse pets after adopter profile has been created.

### PR DESCRIPTION
**Pull request by egemen-dev**
### Describe your change in plain English.

Redirects the user to browse pets after the adopter profile has been created.

### Link to the issue
https://github.com/rubyforgood/pet-rescue/issues/98

### CODE REVIEW

File: app/controllers/registrations_controller.rb [hunk 1]
```diff
@@ -35,9 +35,9 @@ def account_update_params
       :current_password)
   end
 
-  # redirect new adopter users to adopter_profile#new
+  # redirect new adopter users to adoptable pets page
   def after_sign_up_path_for(resource)
-    resource.adopter_account ? new_profile_path : root_path
+    resource.adopter_account ? adoptable_pets_path : root_path
```
Comment (egemen-dev) [inline app/controllers/registrations_controller.rb:40]: This is a valid path. But I got some questions.
Are we supposed to create the `adopter_account` from the admin dashboard?
I am not sure where to create the `adopter_account` since `user has_one :adopter_account, dependent: :destroy` and there seems to be no after callback to create an `adopter_account` for the user.

File: app/controllers/registrations_controller.rb [hunk 2]
```diff
@@ -35,9 +35,9 @@ def account_update_params
       :current_password)
   end
 
-  # redirect new adopter users to adopter_profile#new
+  # redirect new adopter users to adoptable pets page
```
Comment (edwinthinks) [inline app/controllers/registrations_controller.rb:38]:
```suggestion
```
I personally think this comment isn't needed

File: app/controllers/registrations_controller.rb (hunk 1 above)
Comment (edwinthinks) [reply app/controllers/registrations_controller.rb:40]: Is the goal of this to make it so only adopters go into the browser page versus the staff? Honestly... I would just point to `adoptable_pets_path` without any logic to try to handle staff & adopter differently. Internally we've talked about coming back to address the staff post login experience.

What ya think?

**Review by edwinthinks — CHANGES_REQUESTED** [review]
Left some questions about this, lemme know if you have questions.

File: app/controllers/registrations_controller.rb (hunk 1 above)
Comment (egemen-dev) [reply app/controllers/registrations_controller.rb:40]: I agree. I will change `resource.adopter_account ? adoptable_pets_path : root_path` to `adoptable_pets_path`

Comment (edwinthinks) [reply app/controllers/registrations_controller.rb:40]: Excellent! Thanks for hearing me out :)

**Review by edwinthinks — APPROVED** [review]
